doc/api/n-api.md

```diff
@@ -6310,6 +6310,147 @@ idempotent.
 
 This API may only be called from the main thread.
 
+## Using embedded Node.js
+
+### `napi_create_platform`
+
+<!-- YAML
+added: REPLACEME
+-->
+
+> Stability: 1 - Experimental
+
+```c
+napi_status napi_create_platform(int argc,
+                                    char** argv,
+                                    int exec_argc,
+                                    char** exec_argv,
+                                    napi_error_message_handler err_handler,
+                                    napi_platform* result);
+```
+
+* `[in] argc`: CLI argument count, pass 0 for autofilling.
+* `[in] argv`: CLI arguments, pass NULL for autofilling.
+* `[in] exec_argc`: Node.js CLI options count.
+* `[in] exec_argv`: Node.js CLI options.
+* `[in] err_handler`: If different than NULL, will be called back with each
+  error message. There can be multiple error messages but the API guarantees
+  that no calls will be made after the `napi_create_platform` has returned.
+  In practice, the implementation of graceful recovery is still in progress,
+  and many errors will be fatal, resulting in an `abort()`.
+* `[out] result`: A `napi_platform` result.
+
+This function must be called once to initialize V8 and Node.js when using as a
+shared library.
+
+### `napi_destroy_platform`
+
+<!-- YAML
+added: REPLACEME
+-->
+
+> Stability: 1 - Experimental
+
+```c
+napi_status napi_destroy_platform(napi_platform platform, int *exit_code);
```

Comment:
`napi_destroy_environment` has an `exit_code` parameter; `napi_destroy_platform` does not.

Reply:
Don't forget to fix the doc for `napi_destroy_environment` also.